ts/src/test/Exchange/base/test.market.ts

```diff
@@ -87,13 +87,6 @@ function testMarket (exchange: Exchange, skippedProperties: object, method: stri
         // otherwise, it must be false or undefined
         testSharedMethods.assertInArray (exchange, skippedProperties, method, market, 'margin', [ false, undefined ]);
     }
-    if (!('contractSize' in skippedProperties)) {
-        if (!market['spot']) {
-            // if not spot, then contractSize should be defined
-            assert (market['contractSize'] !== undefined, '"contractSize" must be defined when "spot" is false' + logText);
-        }
-        testSharedMethods.assertGreater (exchange, skippedProperties, method, market, 'contractSize', '0');
-    }
     // typical values
```

Comment:
this block was slightly repeated, it is checked down below in other part, in more predictable way.